### ISSUE TITLE
Virtua Fighter 4 - Evolution (Japan) (SLPM-65270) small Health bar inaccuracies

### ISSUE THREAD

**Issue by Thesch28** [PCSX2/pcsx2#3107]
PCSX2 version:
v1.5.0-dev-3281-g877c4358d

PCSX2 options:
all default, tested on dx11 and opengl both in Hardware and Software mode, native resolution,

Description of the issue:
The normal and VF1 health bar present some minor inaccuracies (some parts of the texture not being merged)
These issues also appear in the US version, the VF1 health bar has the same issue in Virtua Fighter 10th Anniversary.

I highlighted the issues in light blue.
(normal health bar)
![image](https://user-images.githubusercontent.com/25397223/64500988-d1d9cb80-d27c-11e9-92e4-2a0c2b2c0cc1.png)
(VF1 Health Bar)
![image](https://user-images.githubusercontent.com/25397223/64501093-2c732780-d27d-11e9-81c4-a5631f0e3b4e.png)
(How they're supposed to look like)
![image](https://user-images.githubusercontent.com/25397223/64501133-4c0a5000-d27d-11e9-89a4-a81a20f81384.png)

How to reproduce the issue: 
Just play any game mode, for the VF1 Health bar both plays must use VF1 Skins in VS mode by holding Triangle+Circle immediately after selecting the stage until the match starts.

Memory Card with a VF4Evo JP save file with VF1 skins unlocked for all characters [VF4Evo JP Save.zip](https://github.com/PCSX2/pcsx2/files/3588558/VF4Evo.JP.Save.zip)

PC specifications: Win7x86/i5-3470/ GeForce GT 520

**Comment by bositman** [comment]
Try playing around with texture filtering see if it affects this

**Comment by Thesch28** [comment]
I tried Nearest, Bilinear (Forced Excluding Sprite), Bilinear (PS2) and the health bars still have those issues.
Bilinear (forced) makes things worse and also makes the whole image blurry, Here's an image of that:
![image](https://user-images.githubusercontent.com/25397223/64575708-69e5bc80-d332-11e9-9398-5ecedffb4c2e.png)

**Comment by Silanda** [comment]
PAL version is the same, as you might expect. I've actually got a couple of games that have texture misalignment problems in software mode like this. Assault Suits Valken is another one (only have test PAL version to test).

**Comment by MrCK1** [comment]
Can one of you upload a GS Dump for this?

**Comment by Silanda** [comment]
Here's a single frame dump from the PAL version: 
[VF4EvoDump.zip](https://github.com/PCSX2/pcsx2/files/3657877/VF4EvoDump.zip)

**Comment by Thesch28** [comment]
Here's two single frame GS dumps of the Japanese version, one for the normal health bar and another one for the VF1 Health Bar.
[VF4Evo Health bar inaccuracies GS Dump.zip](https://github.com/PCSX2/pcsx2/files/3746110/VF4Evo.Health.bar.inaccuracies.GS.Dump.zip)

**Comment by Thesch28** [comment]
Here's two single frame GS Dumps of the Japanese version, one for the normal health bar and the other for the VF1 style health bar.
[VF4Evo Health bar inaccuracies GS Dump.zip](https://github.com/PCSX2/pcsx2/files/3746121/VF4Evo.Health.bar.inaccuracies.GS.Dump.zip)

**Comment by MrCK1** [comment]
Are you sure this isn't bugged on PS2 hardware as well?

**Comment by Thesch28** [comment]
> Are you sure this isn't bugged on PS2 hardware as well?

Yes i'm sure of it, look at these screenshots from PS2 Hardware as you can see it isn't bugged on PS2.
![image](https://user-images.githubusercontent.com/25397223/73229021-f48e5300-413d-11ea-9ea2-9b0bee204bd5.png)

**Comment by MrCK1** [comment]
@Thesch28 @Silanda would you guys be able to retest this on the latest dev build and see if the issue is still active? Thanks.

**Comment by refractionpcsx2** [comment]
Affected by #6553 but still WIP

**Comment by scju** [comment]
It still exists on v1.7.4524
![image](https://github.com/PCSX2/pcsx2/assets/31085634/5fe9ce03-b52e-47aa-960e-6668cce2a42b)
